File: lib/models/api_models/multi_product/multi_product.dart

import 'package:json_annotation/json_annotation.dart';

part 'multi_product.g.dart';

@JsonSerializable()
class MultiProduct{
  final int multiProductId;
  final int parentProductId;
  final String multiProductName;
  final String? multiProductImage;
  final String? multiProductLocalName;
  final String info;

  MultiProduct({required this.multiProductId, required this.parentProductId, required this.multiProductName,required this.multiProductLocalName, required this.multiProductImage, required this.info});

  factory MultiProduct.fromJson(Map<String,dynamic> json) => _$MultiProductFromJson(json);

  Map<String,dynamic> toJson() => _$MultiProductToJson(this);
}

File: android/app/src/main/kotlin/com/gulftechinnovations/android_pos_mini/model/MultiProduct.kt
package com.gulftechinnovations.android_pos_mini.model

data class MultiProduct(
    var multiProductId:Int = 0,
    val parentProductId:Int,
    val multiProductName:String,
    val multiProductLocalName:String? = null,
    val multiProductImage:String?=null,
    val info:String? = null
)